File: 加密保护/WinLicense v2.1.0.10/WinlicenseSDK/ExamplesSDK/EncryptedStrings/Delphi/Unit1.pas
unit Unit1;

interface

uses
  Windows, Messages, SysUtils, Variants, Classes, Graphics, Controls, Forms,
  Dialogs, StdCtrls, WinLicenseSDK;

type
  TForm1 = class(TForm)
    Button1: TButton;
    Button2: TButton;
    Button3: TButton;
    procedure Button1Click(Sender: TObject);
    procedure Button2Click(Sender: TObject);
    procedure Button3Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  Form1: TForm1;

implementation

{$R *.dfm}

procedure TForm1.Button1Click(Sender: TObject);
begin

  // the string inside WLStringDecrypt is encrypted in protection time
  // it will be decrypted when WLStringDecrypt is called

  ShowMessage(WLStringDecrypt('This is an encrypted string 1!'));

end;

procedure TForm1.Button2Click(Sender: TObject);
begin

  // the string inside WLStringDecrypt is encrypted in protection time
  // it will be decrypted when WLStringDecrypt is called

  ShowMessage(WLStringDecrypt('This is an encrypted string 2!'));

end;

procedure TForm1.Button3Click(Sender: TObject);
begin

  // the string inside WLStringDecrypt is encrypted in protection time
  // it will be decrypted when WLStringDecrypt is called

  ShowMessage(WLStringDecrypt('This is an encrypted string 3!'));

end;

end.

File: 加密保护/WinLicense v2.1.0.10/WinlicenseSDK/ExamplesSDK/EncryptedStrings/C/Borland/Unit1.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Unit1.h"
#include "WinlicenseSDK.h"
#pragma comment (lib,"WinlicenseSDK.lib")

//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TForm1 *Form1;
//---------------------------------------------------------------------------
__fastcall TForm1::TForm1(TComponent* Owner)
        : TForm(Owner)
{
}
//---------------------------------------------------------------------------

void __fastcall TForm1::Button1Click(TObject *Sender)
{

  // the string inside WLStringDecrypt is encrypted in protection time
  // it will be decrypted when WLStringDecrypt is called

  ShowMessage(WLStringDecrypt("This is an encrypted string 1!"));

}
//---------------------------------------------------------------------------

void __fastcall TForm1::Button2Click(TObject *Sender)
{

  // the string inside WLStringDecrypt is encrypted in protection time
  // it will be decrypted when WLStringDecrypt is called

  ShowMessage(WLStringDecrypt("This is an encrypted string 2!"));
        
}
//---------------------------------------------------------------------------
void __fastcall TForm1::Button3Click(TObject *Sender)
{

  // the string inside WLStringDecrypt is encrypted in protection time
  // it will be decrypted when WLStringDecrypt is called

  ShowMessage(WLStringDecrypt("This is an encrypted string 3!"));

}
//---------------------------------------------------------------------------
